pkg/operator/sync.go

```diff
@@ -139,6 +145,10 @@ func (optr *Operator) syncAll(syncFuncs []syncFunc) error {
 		return fmt.Errorf("error syncing progressing status: %w", err)
 	}
 
+	if err := optr.syncMachineConfigPoolValidatingWebhook(); err != nil {
```

Comment:
should this live in `syncMachineConfigController` since it's main object is the MCP?